Dockerfile

```diff
@@ -52,3 +52,5 @@ COPY package.json .
 
 USER node
 ENV NODE_ENV=production
+# Prevent npm update check while running any npm run <task>
+ENV NO_UPDATE_NOTIFIER=true
```

Comment:
Great addition! I noticed the update check as well.

Reply:
Thanks, I updated that line as it should be: `npm_config_update_notifier=false`